### DISCUSSION TITLE
bZ4x battery charge during charging

### DISCUSSION THREAD

**Discussion by uphillbattle** [DurgNomis-drol/ha_toyota#140] [General]
While charging a Toyota bZ4x (EV) last night, I noticed that the battery State of Charge did not change while charging. I started charging at 70 % SOC and wanted to charge to 80 %. Every report (when the servers actually responded - they ARE quite flaky, aren't they?) kept saying 70 % until after a few hours when the report suddenly said 98 %...

I'm not sure if this is a server issue or if they car maybe doesn't really report it's SOC charge while charging (maybe it only reports when charging is stopped just like the car doesn't seem to report anything while driving). Has anyone seen this and figure out which it is (server issue or the car not reporting while charging). If it's the latter, I'll pause charging every now and then just to get a SOC update.

**Comment by DurgNomis-drol** [comment]
As far as I know, cars only report information when stopped or properly when charging is done. Unfortunately there is nothing I can do about it

And yes the servers are very flaky 😆

**Reply by uphillbattle** [reply]
I suspected as much. When charging next, I'll make sure to pause the charging for a couple of minutes every half hour to see if the SOC % updates when charging pauses.
